README.md

```diff
@@ -132,3 +132,10 @@ set "JAVA_HOME=%USERPROFILE%\apps\java-1.8.0"
 set "INCLUDE=%INCLUDE%;%OPENSSL_64%\include"
 mvn clean install
 ```
+
+Contributing
```

Comment:
Just to maintain consistency with the Elytron repo, let's use Contributions here instead.